russstation/code/modules/paperwork/pen.dm

```diff
@@ -0,0 +1,17 @@
+
+/obj/item/pen/laughter/attack(mob/living/M, mob/user)
+	if(!istype(M))
+		return
+
+	if(..())
+		if(reagents.total_volume)
+			if(M.reagents)
+
+				reagents.trans_to(M, reagents.total_volume, transfered_by = user, methods = INJECT)
+
+
+/obj/item/pen/laughter/Initialize()
+	. = ..()
+	create_reagents(45, OPENCONTAINER)
```

Comment:
also this should probably be equal to however much chemicals comes in it on initialization so people can't fill it with extra random stuff